go-ssb-bindings/ctrl.go

```diff
@@ -413,7 +413,8 @@ func ssbInviteAccept(token string) bool {
 	ctx, cancel := context.WithCancel(longCtx)
 	err = invite.Redeem(ctx, tok, sbot.KeyPair.Id)
 	defer cancel()
-	if err != nil {
+	if err != nil && !strings.Contains(err.Error(), "already following") {
```

Comment:
strings is undefined (at least it is what my go compiler says)

Reply:
Right, forgot to stage the import, thx!